packages/dtd-catalog-exporter/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "dtd-catalog-exporter-job",
+  "version": "1.0.0",
+  "private": true,
+  "description": "PagoPA Interoperability catalog exporter job",
+  "type": "module",
+  "scripts": {
+    "test": "vitest",
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start:migrate": "node --loader ts-node/esm -r 'dotenv-flow/config' ./src/index.ts",
+    "start:verify": "node --loader ts-node/esm -r 'dotenv-flow/config' ./src/read-models-migration-check.ts",
+    "build": "tsc"
```

Comment:
Missing `check` job that we have in all other services to check TS also for tests files, and the corresponding TSConfig check file

```
    "check": "tsc --project tsconfig.check.json"
```